co.luoja.Telyn.yml

```diff
@@ -0,0 +1,49 @@
+app-id: co.luoja.Telyn
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: TelynGtk
+cleanup:
+  - swift-5.5-RELEASE-ubuntu20.04
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+skip-arches:
+  - aarch64
+modules:
+  - name: swift-lib
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://swift.org/builds/swift-5.5-release/ubuntu2004/swift-5.5-RELEASE/swift-5.5-RELEASE-ubuntu20.04.tar.gz
+        sha256: aaf7eaefdc3f90f77b21b8829546fc2648e02928af476437d6a58556cb858ec6
+    build-commands:
+      - tar -xzf swift-5.5-RELEASE-ubuntu20.04.tar.gz
+      - mkdir -p /app/lib
+      - cp -r swift-5.5-RELEASE-ubuntu20.04/usr/lib/swift/linux/* /app/lib
+  - name: telyngtk
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://gitlab.com/t2056/telyngtk/-/jobs/1645149843/artifacts/raw/TelynGtk-Release.tar.gz
+        sha256: fc23535097fd9523127cebfcf17bf4fefd72cf7abc658371e0b0c690cae3ae08
+    build-commands:
+      - mkdir /app/bin
+      - cp TelynGtk /app/bin/
+      - cp -r *.resources /app/bin/
+  - name: share
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://gitlab.com/t2056/telyngtk.git
+        tag: 1.0.3     
+        commit: 98210f26ffa9f1c22bb3b9d4da42290f3170f8d3
+    build-commands:
+      - install -D co.luoja.Telyn.desktop /app/share/applications/co.luoja.Telyn.desktop
+      - install -D co.luoja.Telyn.svg /app/share/icons/hicolor/scalable/apps/co.luoja.Telyn.svg
+      - install -D co.luoja.Telyn-symbolic.svg /app/share/icons/hicolor/scalable/apps/co.luoja.Telyn-symbolic.svg
+      - install -D co.luoja.Telyn.appdata.xml /app/share/metainfo/co.luoja.Telyn.appdata.xml
```

Comment:
It would probably make sense to include these files in the tar.gz you are using above.